src/main.cpp

```diff
@@ -189,19 +210,53 @@ void setJsonPathPower(JsonDocument json) {
 
 void GetDeviceInfo() {
   JsonDocument jsonResponse;
+  jsonResponse["name"] = shelly_name;

```

Comment:
bitte wieder auf main branch anpassen

Reply:
Mit dieser Info weiß ich gerade nichts anzufangen. Im main branch ist dies ebenso
https://github.com/TheRealMoeder/Energy2Shelly_ESP/blob/main/src/main.cpp#L203-L205 oder wir meinen beide etwas anderes ;-)

Reply:
Stimmt. aber ohne ein rebase wird das so nicht gehen, dafür sind es auch insgesamt viel zu viele Änderungen in einem branch, ist ja fast ein kompletter rewrite...